axelor-stock/src/main/resources/views/StockLocation.xml

```diff
@@ -240,6 +244,12 @@
       method="setStocklocationValue"/>
   </action-method>
 
+  <action-record name="action-stock-location-record-set-stock-location-value-Btn"
```

Comment:
please rename "action-stock-location-record-set-stock-location-value-Btn" => "action-stock-location-record-set-stock-location-value-btn" to follow our convention